e2e/cypress/tests/integration/enterprise/guest_accounts/guest_feature_spec.ts

```diff
@@ -35,7 +35,7 @@ describe('Guest Accounts', () => {
         // # Click "Save".
         cy.get('#saveSetting').then((btn) => {
             if (btn.is(':enabled')) {
-                btn.on('click');
+                btn.on('click', () => {});
```

Comment:
Not to sure on the best approach for this one.

See JQuery.on Docs: https://api.jquery.com/on/